java/arcs/android/sdk/host/ArcHostHelper.kt

```diff
@@ -360,3 +393,20 @@ fun Plan.Partition.createStopArcHostIntent(service: ComponentName, hostId: Strin
         hostId,
         this.toParcelable()
     )
+
+/**
+ * Creates an [Intent] to invoke [ArcHost.setOnArcStateChange] on a [Service]'s internal [ArcHost].
+ */
+fun ComponentName.createOnArcStateChangeIntent(
+    hostId: String,
+    arcId: ArcId,
+    receiver: ResultReceiver
+): Intent =
```

Comment:
Nit: function expressions should fit on one line, so can you make this a return statement instead?

```
): Intent {
  return .....
}
```